examples/metadata.go

```diff
@@ -25,6 +25,16 @@ func MetadataExample() {
 		Location: "https://identity-provider.com/saml/post/logout",
 	})
 
+	idp.AddContactPerson(saml.MetadataBinding{
+		Binding:  saml.HTTPPostBinding,
+		Location: "https://identity-provider.com/saml/post",
+	})
+
+	idp.AddOrganization(saml.MetadataBinding{
```

Comment:
@sccalabrsame for  AddOrganizationfunction should add Organization not MetadataBinding

Reply:
Would that look something like this idp.AddOrganization(saml.Organization{
		OrganizationName:  "OrgName",
		OrganizationDisplayName: "OrgDisplayName",
		OrganizationURL: "OrgUrlName",
	})